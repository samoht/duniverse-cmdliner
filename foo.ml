open Cmdliner

let run _ = ()

let x = Arg.(value & opt string "$(mname)" & info ["arg"])

let t = Term.(const run $ x)

let () = Term.exit @@ Term.eval (t, Term.info "foo")
