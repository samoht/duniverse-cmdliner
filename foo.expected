NAME
       foo

SYNOPSIS
       foo [OPTION]... 

OPTIONS
       --arg=VAL (absent=foo)

       --help[=FMT] (default=auto)
           Show this help in format FMT. The value FMT must be one of `auto',
           `pager', `groff' or `plain'. With `auto', the format is `pager` or
           `plain' whenever the TERM env var is `dumb' or undefined.

